web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -72,13 +73,26 @@
 	// Progress should be reset when the current memory or asset changes.
 	$: memoryIndex, assetIndex, reset();
 
+	const handleKeyDown = (e: KeyboardEvent) => {
```

Comment:
This will need `preventDefault`, or it can be done automatically with `|preventDefault` magic in my other comment.

Reply:
Unfortunately `|preventDefault` cannot be used as it prevents everything, including e.g. `CTRL + R`.